pkg/install/convertconfig.go

```diff
@@ -66,7 +66,15 @@ func GenerateInstallConfig(cd *hivev1.ClusterDeployment, sshKey, pullSecret stri
 	}
 
 	machinePools := []types.MachinePool{}
-	for _, mp := range spec.Machines {
+
+	// combinedMachinePools contains spec.ControlPlane and spec.Compute MachinePools
+	combinedMachinePools := []hivev1.MachinePool{}
+	combinedMachinePools = append(combinedMachinePools, spec.ControlPlane)
+	for _, mp := range spec.Compute {
+		combinedMachinePools = append(combinedMachinePools, mp)
+	}
+
+	for _, mp := range combinedMachinePools {
```

Comment:
Can you make sure the master pool goes in with name "master" for install config here. Technically users could set to whatever they want on our side, but we need to force to masters here.

Reply:
Would be a good unit test too.